src/main/java/edu/cornell/kfs/module/purap/document/authorization/IWantDocumentAuthorizer.java

```diff
@@ -76,6 +76,10 @@ public Set<String> getEditModes(Document document, Person user, Set<String> edit
             result.remove(CUPurapConstants.IWNT_DOC_EDIT_CONTRACT_INDICATOR);
         }
         
+        if(!isInPurchasingAssistantNode(document)) {
+            result.remove(CUPurapConstants.IWNT_DOC_RETURN_TO_SSC);
+        }
+        
```

Comment:
Good question. Initially both the node and the role were called Purchasing Assistant. However, since implementation, the functional users have changed their mind about the role name and they have updated it. I did not update the node name as well. Please let me know if you think it might be good to update the node name too. Thank you.

Reply:
Per Cathy's feedback, I will create a separate user story to update the route node name.

Reply:
I created KFSPTS-32158.